starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,66 @@
 // Names and Input
+var hacker1 = "alex";
+console.log("the driver's name is " + hacker1);
 
+var hacker2 = window.prompt("what is the driver's name?", "name")
+console.log("the navigator's name is " + hacker2);
 
```

Comment:
No need to add "name" in your prompt. Except if it is for User Experience purposes ;)